src/views/mainMenu/MainMenu.vue

```diff
@@ -1,17 +1,70 @@
 <template>
-<div>
-    Main menu
+<div class="main-menu">
+    <div class="main-menu__list">
+        <MainMenuListItem 
+            v-for="(page, index) in pages"
+            :key="index"
```

Comment:
index nie nadaje się do ':key'. 
Lepiej mieć w obiekcie 'page' unikalne id
Sprawdź:
https://vuejs.org/v2/guide/list.html#key

Reply:
W twoim przypadku możesz użyć:
```
:key="page.path"
```
Gdyż ten atrybut musi być unikatowy

Reply:
Ogólnie `key` jest unikalne - każdy element ma inny index. Zrobiłem tak, poważ to statyczna lista. 
Ale fakt - `page.path` będzie bardziej unikalne 😉

Reply:
Key ma być unikalny dla danego elementu listy. 
Index nie jest.
Wystarczy np usunąć element z początku listy. Obiekty się nie zmienia, indexy już tak.